vllm/v1/executor/multiproc_executor.py

```diff
@@ -401,5 +411,8 @@ def worker_busy_loop(self):
                 logger.exception("WorkerProc hit an exception: %s", exc_info=e)
                 continue
 
+            if i == 10 and self.rank == 0:
+                raise ValueError
+            i += 1
```

Comment:
NOTE for reviewer: this is a simple POC for testing all this works with TP>1

Will remove for landing.